components/profile-picture/ProfilePicture.tsx

```diff
@@ -0,0 +1,102 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useEffect, useState } from "react";
+import { StyleSheet, View, Image, Pressable, Animated } from "react-native";
+import { LinearGradient } from "expo-linear-gradient";
+import * as Animatable from "react-native-animatable";
+
+type ProfilePictureProps = {
+  imageSize: number;
+  borderColor: string[];
+  openStories: () => void;
+};
+
+const calculateGradientSize = (imageSize: number): number =>
+  imageSize < 60 ? imageSize + 5 : imageSize + 12;
+
+const calculateInnerCircleSize = (imageSize: number): number =>
+  imageSize < 60 ? imageSize + 2 : imageSize + 7;
```

Comment:
What do you think about making it proportional ?

Reply:
Can you elaborate how yo would?

Reply:
Something like `imageSize * 1.01`.
I don't know if this would be a good approach tho. Just bringing up the idea.